instrumentation/instaamqp091/go.mod

```diff
@@ -1,17 +1,10 @@
 module github.com/instana/go-sensor/instrumentation/instaamqp091
 
-go 1.18
+go 1.16
```

Comment:
Query: Why did we move back to go 1.16?

Reply:
Good catch.
This is a leftover for when I was trying to apply Go workspaces. I will undo the change.